src/system.rs

```diff
@@ -12,32 +12,37 @@ static DEV_OMNECT: &str = "/dev/omnect/";
 static NETWORK_SERVICE: &str = "systemd-networkd.service";
 static NETWORK_SERVICE_RELOAD_TIMEOUT_IN_SECS: u64 = 30;
 
-fn info_current_root() -> Result<()> {
+macro_rules! identity_config_file_path {
+    () => {{
+        const ENV_FILE_PATH_DEFAULT: &'static str = "/mnt/factory/etc/aziot/config.toml";
```

Comment:
that's not necessarily the currently used config file. 
`/etc/aziot/config.toml` is the current used one and can potentially differ.

Reply:
copy paste mistake